blocks/library/text/block.scss

```diff
@@ -0,0 +1,16 @@
+.has-drop-cap {
```

Comment:
We should write this as `p.has-drop-cap`, since modifier classes should never be used without a main class or element as a target.